README.md

```diff
@@ -77,14 +141,14 @@ julia> CodeTracking.whereis(@which uuid1())
 
 ## A few details
 
-CodeTracking won't do anything *useful* unless the user is also running Revise,
+CodeTracking has limited functionality unless the user is also running Revise,
 because Revise will be responsible for updating CodeTracking's internal variables.
 (Using `whereis` as an example, CodeTracking will just return the
 file/line info in the method itself if Revise isn't running.)
 
 However, Revise is a fairly large (and fairly complex) package, and currently it's not
 easy to discover how to extract particular kinds of information from its internal storage.
-CodeTracking is designed to be the new "query" part of Revise.jl.
+CodeTracking is designed to be the "query" part of Revise.jl.
 The aim is to have a very simple API that developers can learn in a few minutes and then
 incorporate into their own packages; its lightweight nature means that they potentially gain
 a lot of functionality without being forced to take a big hit in startup time.
```

Comment:
I feel like this paragraph made sense when CodeTracking was brand-new.
But now I think it is well established as its own thing.
So it should be written as that.
Right now it still feels like it is talking about aspirations.
In particular i think the first sentence about querying revise internals directly can be removed.